crates/nu-command/tests/commands/do_.rs

```diff
@@ -138,7 +139,7 @@ fn ignore_error_with_too_much_stdout_not_hang_nushell() {
         let actual = nu!(
             cwd: dirs.test(), pipeline(
             r#"
-                do -i {sh -c "cat a_large_file.txt"} | complete | get stdout
+                do -c {sh -c "cat a_large_file.txt"} | complete | get stdout
```

Comment:
I think it's better to add three more test cases about `do -c` rather than change existing test

Reply:
I'd agree, except that `-i` no longer captures so these tests won't work with `-i`.